common/flatpak-utils.c

```diff
@@ -6117,3 +6118,68 @@ flatpak_get_cursor_pos (int* row, int *col)
 
   return res == 2;
 }
+
+static void
+print_line_wrapped (int cols, const char *line)
+{
+  g_auto(GStrv) words = g_strsplit (line, " ", 0);
+  int i;
+  int col = 0;
+
+  for (i = 0; words[i]; i++)
+    {
+       int len = g_utf8_strlen (words[i], -1);
+       int space = col > 0;
+
+       if (col + space + len >= cols)
+         {
+           g_print ("\n%s", words[i]);
+           col = len;
+         }
+       else
+         {
+           g_print ("%*s%s", space, "", words[i]);
+           col = col + space + len;
+         }
+    }
+}
+
+void
+flatpak_print_wrapped (int cols,
+                       gboolean bold_red,
+                       const char *text,
+                       ...)
+{
+  va_list args;
+  g_autofree char *msg = NULL;
+  g_auto(GStrv) lines = NULL;
+  int i;
+  const char *on = "";
+  const char *off = "";
+
+  if (bold_red && flatpak_fancy_output ())
+  {
+    on = FLATPAK_ANSI_RED FLATPAK_ANSI_BOLD_ON;
+    off = FLATPAK_ANSI_BOLD_OFF FLATPAK_ANSI_COLOR_RESET;
+  }
+
```

Comment:
I think making a multi-line text bold red is maybe a bit too much.

Is just making the Warning: part red not obvious enough?